internal/cmd/build_test.go

```diff
@@ -239,20 +239,6 @@ func TestBuild(t *testing.T) {
 			},
 		},
 
-		{
-			Name:          "when --num-parallel is nothing",
-			Flags:         []string{"--num-parallel="},
-			Getwd:         defaultWd,
-			ExpectedError: cmd.ErrInvalidNumParallel,
-			SetupMocks: func(m *Mocks) {
-				m.LambgoFileLoader.EXPECT().
-					LoadConfig("/test").
-					Return(&lambgofile.Config{
-						RootPath:   "/some/root/path",
-						BuildPaths: []string{"path1", "path2", "path3"},
-					}, nil)
-			},
-		},
```

Comment:
This is no longer relevant on v3, since the CLI package errors when a flag has no value